frontends/web/src/components/transactions/components/tx-detail-dialog/address-or-tx-id.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2025 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CopyableInput } from '@/components/copy/Copy';
+import styles from './tx-detail-dialog.module.css';
+
+type Props = {
+  values: string[];
+};
+export const AddressOrTxId = ({ values }: Props) => {
+  return (values.map((addrOrTxID) => (
+    <CopyableInput
+      key={addrOrTxID}
+      alignRight
+      borderLess
+      flexibleHeight
```

Comment:
I think flexibleHeight prop is for multi line textareas and not needed here as it should be only 1 line.